code-challenges.rb

```diff
@@ -20,8 +32,14 @@
 # Expected output: 'Sgt Pppr'
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
+ 
 
-
+def removes_all_vowels str
+    newstr = str.delete "aeiouAEIOU"
+    newstr
+end
+    p removes_all_vowels album3
+  
```

Comment:
Looks great. I think you can leave out the `newstr` variable and just let the implicit return handle the logic.